src/TeamDisplay/TeamDisplay.css

```diff
@@ -0,0 +1,42 @@
+.team-section {
+    margin-top: 20px;
+    text-align: center;
+  }
+  
+  .team-section h2 {
+    font-size: 24px;
+    font-weight: bold;
+    margin-bottom: 10px;
+  }
+  
+  .team-grid {
```

Comment:
I know we talked about flex vs grid, let me know if you have any questions while looking at grid implementation